src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, isMarried, sex, partnerName } = person;
+  const manIsMarried = isMarried && sex === 'm';
+  const womanIsMarried = isMarried && sex === 'f';
```

Comment:
Avoid using "magic values":
```js
const MALE = 'm';
const FEMALE = 'f';
```
```suggestion
  const manIsMarried = isMarried && sex === MALE;
  const womanIsMarried = isMarried && sex === FEMALE;
```